.github/workflows/android-ci.yml

```diff
@@ -3,37 +3,33 @@ name: android-ci
 on:
   workflow_dispatch:
   push:
-    branches:
-      - main
-    tags:
-      - 'android-*'
-    paths:
-      - 'platform/android/**'
-      - ".github/workflows/android-ci.yml"
-      - "bin/**"
-      - "expression-test/**"
-      - "include/**"
-      - "metrics/**"
-      - "platform/default/**"
-      - "render-test/**"
-      - "scripts/**"
-      - "src/**"
-      - "test/**"
-      - "vendor/**"
-      - ".gitmodules"
-      - "!**/*.md"
+    # branches:
+    #   - main
+    # tags:
+    #   - 'android-*'
+    # paths:
+    #   - 'platform/android/**'
+    #   - ".github/workflows/android-ci.yml"
+    #   - "bin/**"
+    #   - "expression-test/**"
+    #   - "include/**"
+    #   - "metrics/**"
+    #   - "platform/default/**"
+    #   - "render-test/**"
+    #   - "scripts/**"
+    #   - "src/**"
+    #   - "test/**"
+    #   - "vendor/**"
+    #   - ".gitmodules"
+    #   - "!**/*.md"
 
 jobs:
   build:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-latest
```

Comment:
Even though that it is a container, we decided at some point not to use latest for reproducibility 😊